ci/release/pypi_uploader.sh

```diff
@@ -11,34 +11,30 @@
 
 set -e
 
-
-SOURCE_BRANCH=master
-
 # Skip upload if CI is executed locally
 if [[ ${RUNNING_CI_LOCALLY} = true  ]]; then
     echo "Skipping PyPi upload - running ci locally"
     return 0
-# Skip upload if CI is not executed locally and current branch is not master
-elif [ "${GIT_BRANCH}" != "${SOURCE_BRANCH}" ]; then
-    echo "Skipping PyPI upload - not master branch"
+fi
+
+# Skip upload if CI is not executed locally and current branch is not master or starts with "dev-"
```

Comment:
where is the not executed locally condition check?

Reply:
In the if block above :smiley:

Reply:
got, now it's clear! Thanks :)!